patterns/footer-large.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
- * Title: WooCommerce Large Footer | Light
- * Slug: woocommerce-blocks/footer-large-light
+ * Title: WooCommerce Large Footer
+ * Slug: woocommerce-blocks/footer-large
  * Categories: WooCommerce
  */
 ?>
 
-<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"32px","right":"4%","bottom":"32px","left":"4%"},"blockGap":"40px"},"elements":{"link":{"color":{"text":"var:preset|color|black"}}}},"backgroundColor":"background","textColor":"black"} -->
-<div class="wp-block-group alignfull has-black-color has-background-background-color has-text-color has-background has-link-color" style="padding-top:32px;padding-right:4%;padding-bottom:32px;padding-left:4%"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":"16px"}}} -->
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"32px","right":"4%","bottom":"32px","left":"4%"},"blockGap":"40px"}},"backgroundColor":"background"} -->
```

Comment:
I didn't test, but the issue might be caused by the `backgroundColor` defined here?

Reply:
Thanks for the thorough testing, that was the issue 😅  

Sorry for all the back and forth, hopefully that fixes it 🙏